PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -359,7 +359,7 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
         jetNDauCharged = Var("?userCand('jetForLepJetVar').isNonnull()?userFloat('jetNDauChargedMVASel'):0", "uint8", doc="number of charged daughters of the closest jet"),
     ),
     externalVariables = cms.PSet(
-        mvaTTH = ExtVar(cms.InputTag("electronMVATTH"),float, doc="TTH MVA lepton ID score",precision=14),
+        promptMVA = ExtVar(cms.InputTag("muonPROMPTMVA"),float, doc="Prompt MVA lepton ID score. Corresponds to the previous mvaTTH",precision=14),
```

Comment:
Hello! I think here you forgot to change the name from muonPROMPTMVA to electronPROMPTMVA